cmd/webhook/app/options/globalflags.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"flag"
+	"os"
+
+	"github.com/spf13/pflag"
+
+	"github.com/jetstack/cert-manager/pkg/logs"
+)
+
+func AddGlobalFlags(fs *pflag.FlagSet) {
+	addKlogFlags(fs)
+}
+
+func addKlogFlags(fs *pflag.FlagSet) {
+	local := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```

Comment:
Question: do we aim to add fields for third party flags to the `WebhookConfiguration`? (As I understand in https://docs.google.com/document/d/1FdaEJUEh091qf5B98HM6_8MS764iXrxxigNIdwHYW9c/edit#heading=h.nq4o0th9ianh doing that or not is 'open to discussion'

Reply:
Yes I think we can do - however we'd 'wrap' them ourselves rather than directly exposing third-party structs within our project.

Also, in the klog case, because logging is needed earlier on that config parsing, I've opted to *not* expose these for now. Certainly something we can explore in future if we want.